src/internal_events/internal_metrics.rs

```diff
@@ -0,0 +1,21 @@
+use metrics::counter;
+use vector_core::internal_event::InternalEvent;
+
+#[derive(Debug)]
+pub struct InternalMetricsBytesReceived {
+    pub byte_size: usize,
+}
+
+impl InternalEvent for InternalMetricsBytesReceived {
+    fn emit(self) {
+        trace!(
+            message = "Bytes received.",
+            byte_size = %self.byte_size,
+            protocol = "internal",
```

Comment:
I'm not 100% on the `protocol` here, perhaps it shouldn't be required in the spec?

Reply:
Do you mean that you aren't 100% convinced that `protocol` needs to be a parameter here?
I think I was mimicking `InternalLogsBytesReceived`.

Reply:
Yeah, I'm not sure `internal` makes sense as a "protocol" but I think it's in the spec for these. May be a place to re-evaluate if it's just noise later on.